kedro/framework/cli/starters.py

```diff
@@ -258,7 +260,10 @@ def create_cli():  # pragma: no cover
 @click.option("--checkout", help=CHECKOUT_ARG_HELP)
 @click.option("--directory", help=DIRECTORY_ARG_HELP)
 @click.option("--addons", "-a", "selected_addons", help=ADDON_ARG_HELP)
-def new(config_path, starter_alias, selected_addons, checkout, directory, **kwargs):
+@click.option("--name", "-n", "name", help=NAME_ARG_HELP)
+def new(  # noqa: too-many-arguments
+    config_path, starter_alias, selected_addons, name, checkout, directory, **kwargs
```

Comment:
I didn't notice that `selected_addons` wasn't added as the last argument before `**kwargs` earlier on, but usually new arguments should be added at the end, because adding them somewhere randomly is a breaking change. Of course, all these changes are breaking anyway, but it's good to keep this in mind.